care/facility/summarisation/patient_summary.py

```diff
@@ -0,0 +1,104 @@
+from django.db.models import Q, Subquery
+from django.utils.timezone import now
+
+from celery.decorators import periodic_task
```

Comment:
Codacy found an issue: [Unable to import 'celery.decorators'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5789522)